datadog_checks_base/requirements.in

```diff
@@ -4,6 +4,7 @@ ddtrace==0.13.0
 kubernetes==8.0.1
 prometheus-client==0.3.0
 protobuf==3.7.0
+pyfakefs==3.7.0
```

Comment:
This should instead go in https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/requirements-dev.txt